tutorials/aggregating-minmax/kstreams.html

```diff
@@ -3,5 +3,5 @@
 permalink: /create-stateful-aggregation-minmax/kstreams
 stack: kstreams
 static_data: aggregating-minmax
-help_wanted: true
+ak_javadoc_version: 2.4
```

Comment:
@colinhicks @bbejeck I tried changing this to 2.4 to make my AK docs links render to 2.4 but it doesn't seem to work locally... but maybe will in a staging site?

Reply:
@rspurgeon, try `{{ page.ak_javadoc_version }}`.

Btw, the staging site build command is essentially the same as what we run locally. Generally things should look and behave the same.
https://github.com/confluentinc/kafka-tutorials/blob/master/.semaphore/semaphore.yml#L37

Reply:
{{ page.ak... }} worked!  thanks @colinhicks

Reply:
@colinhicks  One downside to this solution is that if there isn't a value in the page front matter, then it won't default to the site wide value.  That might get confusing if someone copies my solution, maybe not?

Maybe we need something like the jekyll defaults, described here?
https://jekyllrb.com/docs/configuration/front-matter-defaults/

Reply:
oops, the GH api was down when I originally responded to this off-thread:
> yeah, good call. It’s a bit annoying that you can’t simply override `site.` variables with the front matter. I’ll create an issue to set the front matter defaults as you suggest